### PR TITLE
Delay destroying namespace so destroy.sh is not called prematurely

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

**Comment by galderz** [comment]
@ryanemerson CI is failing still.

**Comment by ryanemerson** [comment]
So these changes fix the issues with the eviction tests, however it uncovers that `should_default_cache_be_protected_via_hot_rod` also fails. 

~~I think authentication for REST and hotrod is not working correctly. I will fix as part of this PR and update the tests to ensure that the REST endpoint is properly tested as well.~~

Turns out that authentication was enabled for both Hotrod and REST, but the problem was with the hotrod client configuration.

**Comment by galderz** [comment]
Last build timed out, launching CI again... 

But looking closer, the CI logs shows:

```
Waiting for 30000 seconds before destroying the namespace
```

Waiting for 500 minutes before destroying things?

File: services/functional-tests/src/test/resources/arquillian.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
         <property name="env.setup.script.url">setup.sh</property>
         <property name="env.teardown.script.url">destroy.sh</property>
         <property name="namespace.use.current">true</property>
+        <property name="namespace.destroy.timeout">30000</property>
```
Comment (galderz) [inline services/functional-tests/src/test/resources/arquillian.xml]: 30000 seconds?

**Review by galderz — CHANGES_REQUESTED** [review]

File: services/functional-tests/src/test/resources/arquillian.xml (hunk 1 above)
Comment (ryanemerson) [reply services/functional-tests/src/test/resources/arquillian.xml]: Whoops! I wasn't sure about the timeunit haha

**Comment by ryanemerson** [comment]
@galderz We have a green CI :smiley:

**Comment by ryanemerson** [comment]
@galderz Can we get this merged?

**Review by galderz — APPROVED** [review]

**Comment by galderz** [comment]
Thx @ryanemerson! Nice to see green CI finally :)

**Pull request merged**